### PR TITLE
34 netstat server

### PR DESCRIPTION
**Pull request by realAndrewCohn**
Couple of changes here:
- Created a `/netstat` endpoint that currently just returns the total writes from all of our groups
- Updated `validId.js` which was not checking for the lowest and highest ids and was returning true when it should have been rejecting invalid responses

Things that need to happen next to address #34: create a react client to display some of this D A N K data from our server, thx

### CODE REVIEW

File: code/nexus/package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "server.js",
   "scripts": {
     "test": "npm test",
-    "start": "nodemon server.js"
+    "start": "nodemon server.js",
+    "watch": "supervisor -n index.js --no-restart-on error --inspect index.js"
```
Comment (realAndrewCohn) [inline code/nexus/package.json:9]: also added `npm run watch` to the nexus™️

Refer to [this for more info](https://github.com/DanielSeehausen/fi-field-day/tree/master/code/node-server#useful-resources)

bitmoji

File: code/nexus/package.json [hunk 2]
```diff
@@ -26,6 +27,7 @@
     "express": "^4.16.3",
     "node-fetch": "^2.2.0",
     "nodemon": "^1.18.3",
+    "supervisor": "^0.12.0",
```
Comment (realAndrewCohn) [inline code/nexus/package.json:30]: `npm run watch` (see above) relies on `supervisor`

File: code/node-server/src/middleware/reqValidators/validID.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ const config = require('../../../config.js')
 
 function validID(req) {
   const id = req.query.id
-  return id < config.IDLIMIT['low'] || id > config.IDLIMIT['high'] ? false : true
+  return id <= config.IDLIMIT['low'] || id >= config.IDLIMIT['high'] ? true : false
```
Comment (realAndrewCohn) [inline code/node-server/src/middleware/reqValidators/validID.js:5]: The logic for our `validID` was returning true even when `req.query.id` was undefined. It was also rejecting ids that were on the low or high end of our config. This should fix the issue

File: code/node-server/src/middleware/validator.js [hunk 4]
```diff
@@ -18,6 +18,7 @@ const routeValidators = {
     '/board': [validID],
     '/groups': [validID],
     '/tile': [validTile, validID],
+    '/netstat': [() => true],
```
Comment (realAndrewCohn) [inline code/node-server/src/middleware/validator.js:21]: Not a great validation for the `/netstat` endpoint but needed it in order to make GET requests here

**Comment by realAndrewCohn** [comment]
Partially addresses #34

**Review by DanielSeehausen — APPROVED** [review]
looks great! perhaps move the group total writes counter out of the groups class, but no time to be nitpicky now!

**Pull request merged**